types/three/examples/jsm/controls/ArcballControls.d.ts

```diff
@@ -156,3 +162,5 @@ export class ArcballControls extends EventDispatcher {
 
     dispose(): void;
 }
+
+export {};
```

Comment:
Because the `ArcballControlsEventMap` is not exported.
DefinitelyTyped rules complain that there is a non-exported type. 
This is the way to tell the DefinitelyTyped rule that you want not to export something by default.  
_This is used in many places to allow not export the EventMap_